llvm/lib/Target/PowerPC/PPCRegisterInfo.td

```diff
@@ -155,6 +156,18 @@ foreach Index = 0-31 in {
                 DwarfRegNum<[!add(Index, 32), !add(Index, 32)]>;
 }
 
+let isArtificial = 1 in {
```

Comment:
This isArtificial field looks undocumented in Target.td, can you add some description to it?

Reply:
I more meant it needs docs in Target.td, not just this one use